vcd_cli/disk.py

```diff
@@ -183,6 +183,13 @@ def delete(ctx, name, disk_id):
               required=False,
               metavar='<new-name>',
               help='New name')
+@click.option('iops',
```

Comment:
It seems we have inconsistent usage of @click.option()
Either the first arg is '-abcd' or the 'name of the param'. We should stick to one of the format and follow it across the board.

Reply:
this is the name of the param :)